client/blocks/login/social.jsx

```diff
@@ -188,6 +188,12 @@ class SocialLoginForm extends Component {
 						responseHandler={ this.handleAppleResponse }
 						onClick={ this.trackLogin.bind( null, 'apple' ) }
 					/>
+
+					<p className="login__social-tos">
+						If you continue with Google or Apple and don't already have a WordPress.com account, you
```

Comment:
We'll probably want to localize this right?

Reply:
working on that now.  wanted to get the PR up so it could get Design review

Reply:
Done.